test/sagemaker_tests/tensorflow/tensorflow2_training/integration/sagemaker/test_smmodelparallel.py

```diff
@@ -35,10 +60,8 @@ def test_smmodelparallel_efa(n_virginia_sagemaker_session, efa_instance_type, n_
     """
     Tests SM Modelparallel in sagemaker
     """
-    _, image_framework_version = get_framework_and_version_from_tag(n_virginia_ecr_image)
-    image_cuda_version = get_cuda_version_from_tag(n_virginia_ecr_image)
-    if Version(image_framework_version) < Version("2.4.1") or image_cuda_version != "cu110":
-        pytest.skip("Model Parallelism with EFA only supports CUDA 11, and on TensorFlow 2.4.1 or higher")
+    validate_or_skip_smmodelparallel(n_virginia_ecr_image)
```

Comment:
Why run the less restrictive validation at all when the more restrictive validation for `efa` filters out more cases?